src/DankCord/DankCord.py

```diff
@@ -89,7 +90,7 @@ def _get_command_info(self, name: str) -> dict:
         if self.resource_intensivity == "MEM":
             return self.commands_data.get(name, {})
         else:
-            return json.load(open(f"{self.channel_id}_commands.json")).get(name, {})
+            return json.load(open(f"{self.channel_id}_commands.json", "r+")).get(name, {})
```

Comment:
You don't need to specify the mode? It defaults to read which is the only thing we need when we access the file here.

Reply:
I know, but I added it since type checking was asking for it.